### ISSUE TITLE
How long to train pretrained resnet?

### ISSUE THREAD

**Issue by kuba-chledowski** [kuangliu/pytorch-cifar#24]
Hi! thanks for the repository :)

How long did you train your pretrained resnet 18?

I tried 150, 100, 100 epochs with learning rates decayting as you specified on pre-resnet-50, but I only got ~94.6% accuracy.

**Comment by learning-luke** [comment]
I believe we trained it for 300 epochs with a sinusoidal decaying learning rate, if that helps.
